### PR TITLE
Update messages.es.yml

### PR DESCRIPTION
**Pull request by jami7**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | yes
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.es.yml [hunk 1]
```diff
@@ -321,78 +321,78 @@ import:
...
+         warn_message_2: "Si lo elimine, cada aplicación configurada con ese cliente no se puede ser autorizada por su wallbag."
+         action: 'Quite este cliente'
+    client:
+         page_title: 'Promotor > Nuevo cliente'
+         page_description: 'Va a crear un nuevo cliente. Por favor, llene el campo abajo para URI redirigido de su aplicación.'
+         form:
+             redirect_uris_label: 'los URI redirigidos'
+             save_label: 'Crear un nuevo cliente'
+         action_back: 'Atrás'
+     client_parameter:
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.es.yml]: extra space before `client_parameter:`

**Comment by jami7** [comment]
Does this mean that the extra space has been resolved?

**Comment by j0k3r** [comment]
I've fixed them in my PR

**Comment by jami7** [comment]
Thank you.

**Pull request closed without merging**